src/Plant/water/get_saturated_vapor_pressure.jl

```diff
@@ -0,0 +1,14 @@
+# this function returns the saturated vapor pressure in Pa
+function get_saturated_vapor_pressure(tem::FT) where {FT}
```

Comment:
We should probably use the function in MoistThermodynamics, but we’ll have to wait until it’s been pulled out of ClimateMachine.jl

Reply:
We can add `FT` for type stability